src/server/pfs/server/obj_block_api_server.go

```diff
@@ -299,19 +299,27 @@ func (s *objBlockAPIServer) PutObjectSplit(server pfsclient.ObjectAPI_PutObjectS
 	}
 	var done bool
 	for !done {
+		logrus.Infof("PutObjectSplit 1")
```

Comment:
I think these are some debug logs that got left in?

Disappointed by the lack of swearing tbh

Reply:
Yep yep, cleaned this up before I even saw this comment =P